### DISCUSSION TITLE
Can i join team?

### DISCUSSION THREAD

**Discussion by ThakurRahulSingh** [ROM-builders/temporary#13736] [Q&A]
Please note that i've changed my GitHub username StockAndroid to ThakurRahulSingh, to match my GitHub username and my Telegram username.

**Some successful builds**

1. https://cirrus-ci.com/build/5164504722964480
2. https://cirrus-ci.com/build/5325166388445184
3. https://cirrus-ci.com/build/4626609679368192
4. https://cirrus-ci.com/build/5948980516356096
5. https://cirrus-ci.com/build/5683119490596864
6. https://cirrus-ci.com/build/5612891750531072

cherry picked https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common/commit/af142ae0fac32d3874634c41db5b5b82e3018de3

Here my trees

1. Device Tree                           https://github.com/ThakurRahulSingh/device_xiaomi_miatoll
2. Device Tree-Common           https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common

Here's my GitHub A/C        http://github.com/thakurRahulSingh
Here's my TG A/C               https://t.me/RahulSinghBhadoriya

@Apon77  sir?

**Comment by ThakurRahulSingh** [comment]
@Apon77  bro

**Reply by Apon77** [reply]
please check #team instruction in telegram

**Reply by ThakurRahulSingh** [reply]
now i've cherry-picked something you can check here   https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common/commits/pex-13/doze/AndroidManifest.xml

now am i able to join the team?

@Apon77 please bro

**Reply by ThakurRahulSingh** [reply]
Sorry but i have added wrong link above. i apologies sorry 


cherry picked        https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common/commit/af142ae0fac32d3874634c41db5b5b82e3018de3

**Reply by ThakurRahulSingh** [reply]
@Apon77 bro please check
